packages/grid/rollup.config.js

```diff
@@ -1,57 +1,58 @@
-import resolve from '@rollup/plugin-node-resolve';
 import typescript from 'rollup-plugin-typescript2';
 import cleaner from 'rollup-plugin-cleaner';
 import sourceMaps from 'rollup-plugin-sourcemaps';
 import { terser } from 'rollup-plugin-terser';
 import dts from 'rollup-plugin-dts';
 import command from 'rollup-plugin-command';
-import pkg from './package.json';
+import pkg from './data-grid/package.json';
 
 // dev build if watching, prod build if not
 const production = !process.env.ROLLUP_WATCH;
 export default [
   {
-    input: 'src/index.ts',
+    input: './data-grid/src/index.ts',
     output: [
       {
-        file: 'dist/index-esm.js',
+        file: './data-grid/dist/index-esm.js',
         format: 'esm',
         sourcemap: !production,
       },
       {
-        file: 'dist/index-cjs.js',
+        file: './data-grid/dist/index-cjs.js',
         format: 'cjs',
         sourcemap: !production,
       },
     ],
 
     external: [...Object.keys(pkg.peerDependencies || {})],
     plugins: [
-      resolve({
-        resolveOnly: [/^@material-ui\/x-.*$/], // we bundle x-license and x-grid-modules
-      }),
       production &&
         cleaner({
-          targets: ['./dist/'],
+          targets: ['./data-grid/dist/'],
         }),
-      typescript({ tsconfig: 'tsconfig.build.json' }),
+      typescript({ tsconfig: 'tsconfig.json' }),
```

Comment:
I understand how the 2 files are used.
Not sure we need the one for linting.

Reply:
Why wouldn't need one for linting,

Reply:
because at the root you specify to lint every files with ts js tsx extensions, with no-emit, and it uses the eslint ignore
same inside packages
the only place it is used it's for the yarn typescript command which is just used to check the test as we exclude them in the build

Reply:
Sorry, I wasn't clear. There are two linting stages: eslint and TypeScript. These are complementary. I was referring to the latter.

Reply:
From what I know:

- lint with eslint is configured at the root and ran with a single command.
- lint with TypeScript is run in parallel with Lerna and defined in each package.

Reply:
> the only place it is used it's for the yarn typescript command which is just used to check the test as we exclude them in the build

`yarn typescript` tests: all the demos of the documentation, all the end-to-end tests, all the TypeScript tests. Without this command, we would get silent fails (not reported in the CI).